web/client/main.js

```diff
@@ -155,17 +179,35 @@ function start(isInitiator) {
             pc.createOffer(localDescCreated, logError);
     };
 
+    // start the chat
+    if (chatCheckBox.checked) {
+    if (isInitiator) {
+
+        if(pc.createDataChannel === undefined) {alert("Sorry, please use a browser with datachannel support.");}
```

Comment:
I think a better solution is to disable the checkbox initially if the browser does not support DC.